src/configuration/config_crypter.go

```diff
@@ -125,43 +139,47 @@ func decryptConfig(key, value string) string {
 
 // EncryptSimple string to base64 crypto using AES
 // - Concept from: https://gist.github.com/manishtpatel/8222606
-func encryptSimple(key []byte, text string) string {
+func encryptSimple(key []byte, text string) (string, error) {
 	// key := []byte(keyText)
 	plaintext := []byte(text)
 
 	block, err := aes.NewCipher(key)
 	if err != nil {
-		panic(err)
+		log.Println("ENCRYPT: Failed to encrypt: " + err.Error())
+		return "", err
 	}
 
 	// The IV needs to be unique, but not secure. Therefore it's common to
 	// include it at the beginning of the ciphertext.
 	ciphertext := make([]byte, aes.BlockSize+len(plaintext))
 	iv := ciphertext[:aes.BlockSize]
 	if _, err := io.ReadFull(rand.Reader, iv); err != nil {
-		panic(err)
+		log.Println("ENCRYPT: Failed to encrypt: " + err.Error())
```

Comment:
Should this be Failed to decrypt? Or could it be a slightly different text than line 148?

Reply:
It's inside the encryptSimple() function, so I should think not?